framework/auth/views.py

```diff
@@ -35,6 +36,21 @@
 from osf.utils.requests import check_select_for_update
 from osf import features
 
+
+def throttle_password_reset(user):
+    if not throttle_period_expired(user.submit_reset_token_last_attempt, settings.TIME_RESET_SUBMIT_PASSWORD_TOKEN):
+        last_attempt = user.submit_reset_token_last_attempt.replace(tzinfo=pytz.utc)
+        time_since_throttle = (timezone.now() - last_attempt).total_seconds()
+        wait_time = int(settings.TIME_RESET_SUBMIT_PASSWORD_TOKEN - time_since_throttle)
+        error_data = {
+            'message_short': 'Too Many Requests.',
+            'message_long': 'The you have entered an incorrect token, please wait {} seconds to retry.'.format(wait_time)
```

Comment:
Remove the "The" -- "You have entered an incorrect token, please wait {} seconds to retry."